libraries/AP_Airspeed/AP_Airspeed.cpp

```diff
@@ -489,7 +489,35 @@ void AP_Airspeed::init()
             num_sensors = i+1;
         }
     }
+
+#if AP_AIRSPEED_UAVCAN_ENABLED
+    // we need a 2nd pass for DroneCAN sensors so we can match order by DEVID
+    // the 2nd pass accepts any devid
+    for (uint8_t i=0; i<AIRSPEED_MAX_SENSORS; i++) {
+        if (sensor[i] == nullptr && (enum airspeed_type)param[i].type.get() == TYPE_UAVCAN) {
+            sensor[i] = AP_Airspeed_UAVCAN::probe(*this, i, 0);
+            if (sensor[i] && !sensor[i]->init()) {
```

Comment:
init currently always returns true for the UAVCAN backend. So this if will never trigger.

Reply:
right, we can remove

Reply:
removed